content/source/docs/enterprise/users-teams-organizations/organizations.html.md

```diff
@@ -77,7 +77,7 @@ The authentication page allows owners to determine when users must reauthenticat
 
 The SSH keys page manages any keys necessary for cloning Git-based module sources during Terraform runs. It does not manage keys used for accessing a connected VCS provider. See [SSH Keys for Cloning Modules](../workspaces/ssh-keys.html) for more information.
 
-### Sentinel Policies
+### Policies and Policy Sets
```

Comment:
The next para reads a little weird with Sentinel removed from this header. I'd suggest either restoring it, or updating the para to focus a little more on policies and secondarily on what it's called.